src/System.Private.CoreLib/shared/System/DateTime.cs

```diff
@@ -1182,11 +1188,12 @@ public int Year
         // Checks whether a given year is a leap year. This method returns true if
         // year is a leap year, or false if not.
         //
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static bool IsLeapYear(int year)
         {
             if (year < 1 || year > 9999)
```

Comment:
```suggestion
             if ((uint)(year - 1) > (9999 - 1))
```

Reply:
thanks for the suggestion but I think that it would rather not increase the perf but decrease the readability of the source code